### ISSUE TITLE
rinkeby/ropsten forking doesnt work

### ISSUE THREAD

**Issue by hashfabric** [nomiclabs/hardhat#1155]
Trying to deploy `Greeter.sol` using  `npx hardhat node --fork https://rinkeby.infura.io/v3/<project_id>`
```
$ npx hardhat run scripts/sample-script.js 
ProviderError: Transaction failed: revert
    at HttpProvider.request (/Users/h/code/hhtest/node_modules/hardhat/src/internal/core/providers/http.ts:45:19)
    at GanacheGasMultiplierProvider.request (/Users/h/code/hhtest/node_modules/hardhat/src/internal/core/providers/gas-providers.ts:170:34)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```
Node logs:
```
hardhat_addCompilationResult
web3_clientVersion (2)
eth_chainId
eth_accounts
eth_chainId
eth_estimateGas
  Contract deployment: Greeter
  From:                0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
  Value:               0 ETH

  Transaction failed: revert
```
Works fine with mainnet forking.

**Comment by fvictorio** [comment]
Can confirm :+1:

**Comment by ejwessel** [comment]
I actually have the reverse. It works on Rinkeby but not Mainnet.

**Comment by fvictorio** [comment]
We'll release a fix for this soon. For the time being, changing the default mnemonic is a temporary workaround.

**Comment by fvictorio** [comment]
This was fixed in 2.0.10.
